integration_tests/pyproject.toml

```diff
@@ -17,9 +17,11 @@ protobuf = "^3.13.0"
 grpcio = "^1.33.2"
 PyYAML = "^5.3.1"
 python-dateutil = "^2.8.1"
-pystarport = { git = "https://github.com/crypto-com/pystarport.git", branch = "main" }
 web3 = "^5.20.1"
 eth-bloom = "^1.0.4"
+python-dotenv = "^0.19.2"
+deepdiff = "^5.6.0"
+pystarport = {git = "https://github.com/damoncro/pystarport.git", rev = "feature/expansion"}
```

Comment:
Should change to crypto-com repo after the corresponding PR merged.

Reply:
Done

Reply:
`python-dotenv` is used in `integration_tests/utils.py`, should be kept.